### DISCUSSION TITLE
Link to search results?

### DISCUSSION THREAD

**Discussion by eagereyes** [vuejs/vitepress#3229] [Q&A] [answered]
Is it possible to create a link to a page with the search modal open? I figure this would be done with a URL parameter like, `?query=keyword` or similar. Is this possible at all?

**Comment by brc-dd** [comment, marked as answer]
We are thinking of doing like #2983 instead -- most likely a whole page when `/search?query=keyword` is visited. Please track that instead and comment there if you've any suggestions.

**Comment by eagereyes** [comment]
That sounds good! Thanks!
